### PR TITLE
Added an entry to the README on how to add a helm chart to the mono repo

### PR DESCRIPTION
**Pull request by jaxels10**
**Description of your changes:**
Added an entry to the README on how to add a helm chart to the mono repo

Checklist:

* [ ] I have bumped the chart version
* [x] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [ ] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
* [ ] I have created the necessary tests in the chart, if they are possible/necessary
* [ ] I have squashed commits if necessary

### CODE REVIEW

**Comment by skovbakke** [comment]
@jacobcrawford should be testing this very soon

File: README.md [hunk 1]
```diff
@@ -1,10 +1,23 @@
 # helm-charts
 This is our mono repository that contains all of our proxy charts. 
 
+### Adding a Helm chart
+If you would like to add a new service or Helm chart in general to this repository, please follow these steps carefully. 
+1. Add a new folder to the charts/ directory named after your Helm chart. 
+2. Inside this new folder, add a chart/ folder and put your Helm chart in there
+3. It is important that you have a values.yaml file, even if it is empty, since the linter will otherwise fail
```
Comment (jacobcrawford) [inline README.md:8]: I think we should also have a README.md file to describe the chart.

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
Nothing much.

File: README.md (hunk 1 above)
Comment (jaxels10) [reply README.md]: I have now added this and when the workflows are (hopefully) passed, it is ready for re-review :)

**Review by jacobcrawford — APPROVED** [review]
LGTM

**Pull request merged**